### ISSUE TITLE
Remove Docker usage in Travis-CI

### ISSUE THREAD

**Issue by PMeira** [dss-extensions/dss_capi#39]
After #25, we only need Free Pascal to build DSS C-API -- on x64 (Linux x86 still needs Docker).

This is low priority.

**Comment by PMeira** [comment]
I'll leave a branch `dockerless` if it comes handy later. Right now SourceForge is having connectivity issues while the Docker Hub download works fine, so there's no point in removing Docker if it would make things worse.
